ml-pipelines/README.md

```diff
@@ -46,6 +46,15 @@ This directory contains artifacts for deploying all backend components of ML Pip
 
 This directory contains the service monitor definition for ML Pipelines. It is always deployed by base, so this will eventually be moved into the base directory itself.
 
+## Parameters
+
+You can customize the ML Pipelines deployment by injecting custom parameters to change the default deployment. The following parameters can be used:
+
+* **pipeline_install_configuration**: The ConfigMap name that contains the values to install the ML Pipelines environment. This parameter defaults to `pipeline-install-config` and you can find an example in the [repository](https://github.com/opendatahub-io/odh-manifests/blob/master/ml-pipelines/base/configmaps/pipeline-install-config.yaml).
+* **ml_pipelines_configuration**: The ConfigMap name that contains the values to integrate ML Pipelines with the underlying components (Database and Object Store). This parameter defaults to `kfp-tekton-config` and you can find an example in the [repository](https://github.com/opendatahub-io/odh-manifests/blob/master/ml-pipelines/base/configmaps/kfp-tekton-config.yaml).
```

Comment:
To prevent a broken link if this manifest is moved around, these links should be converted to a relative path